uisdk/src/main/java/com/karhoo/uisdk/analytics/Analytics.kt

```diff
@@ -12,83 +12,84 @@ import java.util.Date
 @Suppress("TooManyFunctions")
 interface Analytics {
 
+    @Deprecated("ToBeDeleted")
```

Comment:
You can remove the deprecated method. We don't need to keep them. 
We will communicate with effected partners